frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.scss

```diff
@@ -130,8 +140,15 @@
         display: flex;
         justify-content: space-between;
         overflow: hidden;
+        color: black;
+
+        .LemonIcon {
+            color: var(--muted);
+        }
```

Comment:
I guess it's not possible to add this and the font weight below directly on the affected components?